<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>波普特酒店温控系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../img.qrc">
    <normaloff>:/icon/images/xiaohui.png</normaloff>:/icon/images/xiaohui.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family:century gothic
}

#Form{
	background:url(:/picture/images/background.jpg);
}

QFrame
{
	background:#333;
	border-radius:15px;
}

QToolButton
{

background:red;
border-radius:60px;
}

QLabel
{
	color:white;
}

QPushButton
{
background:red;
color:white;
border-radius:15px;
}

QPushButton:hover
{

color:red;
border-radius:15px;
background:#49ebff;
}

QLineEdit
{

background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}		

QStackedWidget
{

background-color:#333;
}		

QTableWidget::item
{
color:white;
background:#333;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>43</y>
     <width>750</width>
     <height>469</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_left">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_central">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>29</y>
         <width>124</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Central Ctrl</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_branch">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>94</y>
         <width>124</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Room Monitor</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_return">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>423</y>
         <width>84</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_right">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>18</y>
         <width>603</width>
         <height>447</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#333;
</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_central">
        <widget class="QPushButton" name="pushButton_onoff">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>18</y>
           <width>95</width>
           <height>37</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:red;</string>
         </property>
         <property name="text">
          <string>ON/OFF</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_mode">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>76</y>
           <width>233</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <item>
          <property name="text">
           <string>Click to select mode</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cold</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Warm</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_mode">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>80</y>
           <width>157</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Working Mode:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_range">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>145</y>
           <width>215</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Temperature Range:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_min">
         <property name="geometry">
          <rect>
           <x>255</x>
           <y>148</y>
           <width>30</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_temp">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>150</y>
           <width>40</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>℃ -</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_temp2">
         <property name="geometry">
          <rect>
           <x>364</x>
           <y>150</y>
           <width>21</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>℃</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_max">
         <property name="geometry">
          <rect>
           <x>329</x>
           <y>148</y>
           <width>30</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_price_2">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>242</y>
           <width>90</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>min/℃</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_setprice">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>195</y>
           <width>106</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Fee Rate:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_low">
         <property name="geometry">
          <rect>
           <x>214</x>
           <y>240</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_low">
         <property name="geometry">
          <rect>
           <x>36</x>
           <y>239</y>
           <width>185</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Low Wind Speed：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_middle">
         <property name="geometry">
          <rect>
           <x>249</x>
           <y>281</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_price2">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>283</y>
           <width>94</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>min/℃</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_middle">
         <property name="geometry">
          <rect>
           <x>36</x>
           <y>280</y>
           <width>216</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Medium Wind Speed：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_high">
         <property name="geometry">
          <rect>
           <x>36</x>
           <y>317</y>
           <width>186</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>High Wind Speed：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_high">
         <property name="geometry">
          <rect>
           <x>214</x>
           <y>318</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_price3">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>320</y>
           <width>92</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>min/℃</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_fee">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>200</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_price_3">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>200</y>
           <width>90</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>yuan/℃</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_branch">
        <widget class="QTableWidget" name="tableWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>581</width>
           <height>431</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableWidget::item
{
color:white;
background:#333;
}

QHeaderView::section {
    background:#333; /* 设置表头的背景颜色 */
    color: white;           /* 设置表头文本颜色 */
 

}

QTableCornerButton::section {
    background-color: #333; /* 设置左上角表头的背景颜色 */
    color: white;           /* 设置左上角表头的文本颜色 */
}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections/>
</ui>
